src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -113,12 +114,21 @@ public void save(AddressBook addressBook) throws StorageOperationException {
 
     /**
      * Loads data from this storage file.
-     *
+     * 
+     * @return an {@link AddressBook} containing the data in the file, or an empty {@link AddressBook} if it
```

Comment:
Maybe can reword it to `@return the existing {@link AddressBook} if it exists. Else, returns an empty {@link AddressBook}`